snap/snapcraft.yaml

```diff
@@ -45,6 +76,11 @@ parts:
       # Don't overly optimize for build CPU-- stay generic
       - -DBINARY_PACKAGE_BUILD=On
 
+      # Not sure why staged data isn't available to CMAKE by default
+      # is there a better way to include this
```

Comment:
Depends on how darktable finds it. I believe the staging area is added to the module path, but lensfun doesn't seem to include one of those.

Reply:
> I believe the staging area is added to the module path, but lensfun doesn't seem to include one of those.
 
Having trouble parsing that.  After the lensfun "install" step the "stage" area does indeed contain liblensfun (in the arch subdirectory).  However darktable is finding all of its build depends in (not sure of the name) the local environment populated with the build-packages.

I'm not cmake literate enough to know how to interrogate what path is being inspected - would be nice if you could get snapcraft to dump out all of the commands as executed so you could see what the implicit cmake variables were.

Reply:
I mean lensfun doesn't ship a cmake module to support finding itself via cmake (not unusual), so the environment set by snapcraft to find those in the staging area isn't helpful. Snapcraft also does pkg-config magic, but that doesn't appear to be used here either. Your solution is perfectly acceptable, although I suspect if you use the triplet path here you won't need to do the `ln` hack above.